src/Entity/User.php

```diff
@@ -11,68 +11,44 @@
 use Symfony\Component\Serializer\Annotation\Groups;
 use Symfony\Component\Uid\Uuid;
 
-/**
- * @ORM\Entity(repositoryClass=UserRepository::class)
- */
-class User implements UserInterface, LoggableEntityInterface
+#[ORM\Entity(repositoryClass: UserRepository::class)]
+class User implements UserInterface, LoggableEntityInterface, \Stringable
 {
-    /**
-     * @ORM\Id
-     * @ORM\Column(type="uuid", unique=true)
-     */
-    private $id;
+    #[ORM\Id]
+    #[ORM\Column(type: 'uuid', unique: true)]
+    private readonly \Symfony\Component\Uid\UuidV4 $id;
 
-    /**
-     * @ORM\Column(type="string", length=180, unique=true)
-     */
-    private $email;
+    #[ORM\Column(type: 'string', length: 180, unique: true)]
+    private ?string $email = null;
 
-    /**
-     * @ORM\Column(type="json")
-     */
-    private $roles = [];
+    #[ORM\Column(type: 'json')]
+    private array $roles = [];
 
-    /**
-     * @ORM\Column(type="string", length=255, nullable=true)
-     */
+    #[ORM\Column(type: 'string', length: 255, nullable: true)]
     private $loginToken;
 
-    /**
-     * @ORM\Column(type="string", length=255)
-     * @Groups({"mail_template"})
-     */
-    private $name;
+    #[ORM\Column(type: 'string', length: 255)]
+    #[Groups(['mail_template'])]
+    private ?string $name = null;
 
-    /**
-     * @ORM\ManyToOne(targetEntity=Municipality::class)
-     */
-    private $favoriteMunicipality;
+    #[ORM\ManyToOne(targetEntity: Municipality::class)]
+    private ?\App\Entity\Municipality $favoriteMunicipality = null;
 
-    /**
-     * @ORM\OneToMany(targetEntity=CaseEntity::class, mappedBy="assignedTo")
-     */
-    private $assignedCases;
+    #[ORM\OneToMany(targetEntity: CaseEntity::class, mappedBy: 'assignedTo')]
+    private \Doctrine\Common\Collections\ArrayCollection|array $assignedCases;
```

Comment:
This will fail if we try loading fixtures. It should probably say `\Doctrine\Common\Collections\Collection` rather than `\Doctrine\Common\Collections\ArrayCollection|array`.